LiteFarmOrg/LiteFarm
# Lf 2301/invitate existing users in proper language

Fixed a bug where if an existing user was invited to a farm they were not always sent an invitation email matching their language preferences.

To test:
1. Ensure you have at least 2 accounts set up.
2. Make sure that the second account is in a different language than the first.
3. Create a new farm with the first account.
4. Invite the second account to the new farm.
5. The invite email to the second account should show up in the preferred language of the second account, instead of the preferred language of the first.
6. Invite a new user to the new farm (i.e. a user that does not have an account yet)
7. This invite email should be in the preferred language of the person who invited them to the farm.
